filebeat.yml

```diff
@@ -1,4 +1,4 @@
-filebeat.prospectors:
+filebeat.inputs:
```

Comment:
why is this changing?

Reply:
Good question, there is a deprecation warning in the logs: